server/metrics/metrics.go

```diff
@@ -184,6 +201,96 @@ func New(config *libconfig.MetricsConfig) *Metrics {
 				return 0
 			},
 		),
+
+		metricOperationsTotal: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Subsystem: collectors.Subsystem,
+				Name:      collectors.OperationsTotal.String(),
+				Help:      "Cumulative number of CRI-O operations by operation type.",
+			},
+			[]string{"operation_type"},
```

Comment:
```suggestion
			[]string{"operation"},
```
Does this label name really require the `_type` suffix? Same for the other `operations` metrics.

Reply:
We can use just `operation` IMO.

Reply:
Resolved in 563ee1a.